pkg/server/server.go

```diff
@@ -707,13 +708,26 @@ func NewServer(cfg Config, stopper *stop.Stopper) (serverctl.ServerStartupInterf
 	if rangeFeedBudgetFactory != nil {
 		nodeRegistry.AddMetricStruct(rangeFeedBudgetFactory.Metrics())
 	}
+
+	// Create an unlimited (for now) monitor for bytes used by raft entries. Use a
+	// 512 KiB allocation size, so that 96 raft scheduler workers would typically
+	// account as a 48 MiB baseline.
+	raftEntriesMetric := metric.NewGauge(metric.Metadata{
```

Comment:
Should we put this in `nodeMetrics` or something? Should this be per-store instead like the other Raft metrics, with a separate per-store budget pulling from a common node-wide budget or something?

Reply:
Could do. Need to check whether `BytesMonitor` allows more than 2 levels of nesting.

Reply:
It's probably possible, but one thing I to consider is how/when a `BytesMonitor` would return the acquired bytes to the parent pool.

In the current version of this PR, we call `bytesAccount.Clear()` after handling the entries, and this returns all the bytes to the global pool.

If we have something like `Node.RaftEntriesPool -> Store.RaftEntriesPool -> Replica.mu.bytesAccount`, then the bytes will be returned to the `Store` pool. In this case we would need to make sure that this pool sometimes returns bytes to the `Node` pool, otherwise some stores can starve.

Reply:
Added per-store metric too.